duecredit/collector.py

```diff
@@ -219,25 +222,21 @@ def cite(self, entry, **kwargs):
         # TODO: if cite is invoked but no path is provided -- we must figure it out
         # I guess from traceback, otherwise how would we know later to associate it
         # with modules???
+        path = kwargs.get('path', None)
+        if path is None:
+            raise ValueError('path must be provided')
```

Comment:
Probably better? But then if we store version too we would need to make that positional too...what do you think?